server/src/main/java/com/broadcom/lsp/cobol/core/visitor/CobolVisitor.java

```diff
@@ -620,7 +620,7 @@ private void areaAWarning(Token token) {
         .ifPresent(it -> throwException(token.getText(), it, messageService.getMessage("CobolVisitor.AreaAWarningMsg")));
   }
 
-  private void areaBWarning(List<Token> tokenList) {
+  private void areaBWarning(@NonNull List<Token> tokenList) {
```

Comment:
Will throw an exception on `null` tokenList

Reply:
This makes sense. Let's see if Coverity agreed with this.